wand-library-in-python/index.md

```diff
@@ -0,0 +1,182 @@
+from Wand.image import image as wandImage
+# The second step is to import the image using its correct pathname.
+with wandImage(filename='/home/jm/Downloads/python.png') as img:
+# The third step is to print the image's height and width.
+    print('The Height of the image is:', img.height)
+    print('The width ogf the image is:', img.width)
+```
+The input image is:
+
+![python](/engineering-education//home/jm/jm/DESKTOP/jm/projects/article/mukono10/wand-library-in-python/python.png)
+
+The output is as follows.
+```bash
+The height of the image is: 203
+The width of the image is: 601
+```
+### Use Wand package in Python to blur an image
+We can use several ways in Python to blur an image. For example, one can decide to blur an image using the pillow package, another one can abandon the pillow package and use the OpenCV library, and all will perform the same job. However, in our case, we will use the Wand library to blur the same image, and the following are the steps that we shall use.
+
+1. We shall import the image library from Wand as we had done in our previous example.
+```Python
+from Wand.image import Image as wandImage
+```
+2. We shall then import our image using the correct pathname.
+```Python
+with wandImage(filename='/home/jm/Downloads/python.png') as img:
+```
+3. We shall then call the blur function by specifying the radius and sigma of the image.
+
+The radius parameter is used to specify the radius of the gaussian aperture, which is the size of the gap, and it is always an integer number, e.g., 3.
+
+The sigma parameter is used to specify the value of sigma, which is the Gaussian filter's standard deviation, and it is also an integer input, e.g., 5.
+```Python
+img.blur(radius=2, sigma=3)
+```
+4. The last step is to save the image from differentiating it from the original image.
+```Python
+img.save('pathname_image')
+```
+Let us take a look at an example.
+```Python
+# Importing the image library from Wand
+from Wand.image import Image as wandImage
+# Importing the image using the correct image location.
+with wandImage(filename='/home/jm/Downloads/python.png') as img:
+# Blur function and its dimensions    
+    img.blur(radius=1, sigma=3)
+# Saving the blurred image using in a certain location.    
+    img.save(='/home/jm/Downloads/python1.png)
+```
+The input image is:
+
+![python](/engineering-education//home/jm/jm/DESKTOP/jm/projects/article/mukono10/wand-library-in-python/python.png)
+
+The output image is:
+
+![python](/engineering-education//home/jm/jm/DESKTOP/jm/projects/articles/mukono10/wand-library-in-python/python1.png)
+### Image transform using Python wand.
+Another essential and robust feature of the Python wand is the image transform feature. It is used to transform an image to be displayed differently from the display of the original image. The image transformation is achieved through the collaboration of several functions. The following steps are used to transform an image.
```

Comment:
What type of transformation can be done? State them. These statements are unclear.